WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentCardReader.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+struct CardPresentPaymentCardReader {
+    public let name: String
+
+    /// The reader's battery level, if available.
+    /// This is an unformatted percentage as a float, e.g. 0.0-1.0
+    public let batteryLevel: Float?
```

Comment:
nit: I'm guessing these `public` annotations might be from the original intention of keeping them in Yosemite, but since other models in the rest of the PR don't use `public`, do you think we keep them as internal for now?
```suggestion
    let name: String

    /// The reader's battery level, if available.
    /// This is an unformatted percentage as a float, e.g. 0.0-1.0
    let batteryLevel: Float?
```